solidity/depool/README.md

```diff
@@ -624,9 +634,9 @@ DePool balance can be viewed on the [ton.live](https://ton.live/main) blockchain
 tonos-cli account <depool_address>
 ```
 
-Additionally, DePool emits the `TooLowDePoolBalance` event when its balance drops too low, indicating the recommended amount to replenish it.
+Additionally, DePool emits the `TooLowDePoolBalance` [event] when its balance drops too low to perform state update operations.
```

Comment:
It might be a link missing after [event]

Reply:
In the beginninf of the `### DePool Balance` section it may be useful to write that DePool maintains it's pure (without all stakes) balance on the level of balanceThreshold (which was set in the constructor). That's why this parameter is important to be high enough.

Reply:
Also it might be more detailed to write: `when its pure balance drops below CRITICAL_THRESHOLD.`

Reply:
added